src/Model/GContact.php

```diff
@@ -841,7 +841,7 @@ private static function updateFromNoScrape(array $data)
 
 		if ($curlResult->isSuccess() && !empty($curlResult->getBody())) {
 			$noscrape = json_decode($curlResult->getBody(), true);
-			if (!empty($noscrape)) {
+			if (!empty($noscrape) && !empty($noscrape['updated'])) {
```

Comment:
Just ` !empty($noscrape['updated'])` was necessary.

Reply:
Too late :-)

Reply:
There always is a next time you'll have this in mind.